src/globalstyle/globalStyle.js

```diff
@@ -0,0 +1,6 @@
+import React from 'react';
+import './globalStyle.css';
+
+export default function GlobalStyle({ children }) {
+  return <div>{children}</div>;
```

Comment:
Is it not necessary to pass the styling?

Reply:
There are global styles settings, with no classes. The implentation is correct